server/bin/liquibase_wrapper.groovy

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env groovy
+import groovy.grape.Grape
+
+def root = 'buildr -s checkout_root'.execute().text.trim()
+def candlepin_classes_path = "$root/server/target/classes"
+if (!(new File(candlepin_classes_path).exists())) {
+  println('Candlepin classes not found. Please do a build first.')
```

Comment:
If we can detect the files are missing, does it make sense to just do the build here rather than telling the user to do it?